asgs_main.sh

```diff
@@ -1293,6 +1291,7 @@ submitJob()
       DATETIME=`date +'%Y-%h-%d-T%H:%M:%S'%z`
       echo "time.${JOBTYPE}.start : $DATETIME" >> run.properties
       echo "[${DATETIME}] Starting ${JOBTYPE}.${ENSTORM} job in $PWD." >> ${ADVISDIR}/${ENSTORM}/${JOBTYPE}.${ENSTORM}.run.start
+      CPUREQUEST=`expr $NCPU + $NUMWRITERS`
```

Comment:
This looks good.

In `bash` I like to use the following idiom for adding or subtracting since it saves on the creation of a new process,

    CPUREQUEST=$(($NCPU + $NUMWRITERS))